src/foam/java/refinements.js

```diff
@@ -435,10 +435,10 @@ foam.CLASS({
       var m = info.getMethod('cast');
       m.body = 'return ( o instanceof Number ) ?'
             + '((Number)o).intValue() :'
-            + '(int)o;';
+            + '((Integer) o).intValue();';
```

Comment:
Though, this code has been around for a while, not sure why its causing an issue now.